Makefile

```diff
@@ -259,6 +259,15 @@ generate-mocks: mockery ## Generate the mocks for the project as configured in .
 clean-ui:
 	find ${UI_DIR} -type f -name extension.tar -delete
 
+PHONY: codegen-download-openapi
+codegen-download-openapi: run-backend
```

Comment:
This doesn't work as it blocks during `run-backend` and never executes the `yarn` command below

Reply:
add the missing files; ran in local and able to generate the schema.yaml